### PR TITLE
support webpack plugins

### PR DESCRIPTION
**Pull request by kylemac**
- expose + document `plugins` opt for `app.js`
- convert `parseAppJs` to use `babel-core/register`
- test coverage

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -188,15 +186,12 @@ export default class Config {
...
-      const contents = transformFileSync(filename, {
-        filename: filename,
-        presets: [require('babel-preset-es2015'), require('babel-preset-stage-2')]
-      }).code
-      const mod = _eval(contents)
+      const mod = require(filename)
       res = mod.__esModule ? mod.default : mod
     } catch (err) {
-      if (err.code !== 'ENOENT') { throw new Error(err) }
+      if (err.code !== 'MODULE_NOT_FOUND') { throw new Error(err) }
```
Comment (kylemac) [inline lib/config.js]: this'll suppress MODULE_NOT_FOUND errors within someone's `app.js` too. we okay with that? i kinda am

Comment (jenius) [inline lib/config.js]: Hmmm, well we can run an `existsSync` check first instead then it wouldn't surpress it?

Comment (kylemac) [inline lib/config.js]: you mean `fs.statSync` ;) - yeah, i'll do that

Comment (jenius) [inline lib/config.js]: Haha or `fs.accessSync` perhaps

**Comment by kylemac** [comment]
@jenius feedback implemented

**Comment by jenius** [comment]
beautiful. :+1:

**Pull request merged**